airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -279,7 +279,7 @@ def transform_clickhouse(config: Dict[str, Any]):
             "port": config["port"],
             "schema": config["database"],
             "user": config["username"],
-            "password": config["password"],
+            "password": config.get("password", ""),
```

Comment:
This will assign `""` an empty string as the password. Can you check if the expected config is maybe without this key or with None instead?

Reply:
As I see the `password` field may be omitted:
https://github.com/burmecia/dbt-clickhouse/blob/cbac6c11893f8485de417b350c73095192c6ce86/dbt/include/clickhouse/sample_profiles.yml#L13-L19